src/mame/barcrest/mpu4.cpp

```diff
@@ -728,15 +724,14 @@ uint8_t mpu4_state::pia_ic4_portb_r()
 		if (m_optic_pattern & (1<<m_active_reel)) m_ic4_input_b |=  0x08;
 	}
 
-	if ( m_overcurrent )
+	if (m_low_volt_detect)
 	{
-		m_ic4_input_b |= 0x02;
+		if ( m_signal_50hz ) m_ic4_input_b |= 0x04;
 	}
```

Comment:
I wasn’t objecting to the controlled statement on a line of its own – I was objecting to “English” parenthesis spacing `if (m_low_volt_detect)` being followed immediately by “European” parenthesis spacing `if ( m_signal_50hz )` just two lines later.